### ISSUE TITLE
Disable messages

### ISSUE THREAD

**Issue by Marfeld** [ABKAM2023/CS2-RanksPoints#19]
Is there a way to turn off all the messages? It would be nice if you could turn it off.

Greetings Mario

**Comment by mayersgamer** [comment]
i agree i mean at the end of the round 1 message is enough displaying how many point you won or lose
